"Fuzz and run valgrind / build (pull_request)" failing in CI

The "Fuzz and run valgrind / build (pull_request)" test is failing consistently in CI, even on documentation-only changes. See [here](https://github.com/simdjson/simdjson/pull/1205/checks?check_run_id=1215001003) for an example.

The failure is in the "Install packages necessary for building" step, and seems to be related to an offline package source for azure-cli:

```

Get:26 https://apt.llvm.org/bionic llvm-toolchain-bionic-10/main amd64 Packages [8762 B]
Hit:28 http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_18.04  InRelease
Ign:29 https://packages.microsoft.com/repos/azure-cli bionic InRelease
Err:30 https://packages.microsoft.com/ubuntu/18.04/prod bionic InRelease
  Could not connect to packages.microsoft.com:443 (13.90.56.68), connection timed out [IP: 13.90.56.68 443]
Err:31 https://packages.microsoft.com/repos/azure-cli bionic Release
  Unable to connect to packages.microsoft.com:https: [IP: 13.90.56.68 443]
Reading package lists...
E: The repository 'https://packages.microsoft.com/repos/azure-cli bionic Release' no longer has a Release file.
Error: Process completed with exit code 100.
```

--- a/fuzz/build_fuzzer_variants.sh
+++ b/fuzz/build_fuzzer_variants.sh
@@ -45,12 +45,7 @@ variant=sanitizers
 	      -DCMAKE_C_FLAGS="-fsanitize=fuzzer-no-link,address,undefined -fno-sanitize-recover=undefined" \
 	      -DCMAKE_BUILD_TYPE=Debug \
 	      -DSIMDJSON_FUZZ_LINKMAIN=Off \
-<<<<<<< HEAD
 	      -DSIMDJSON_FUZZ_LDFLAGS="-fsanitize=fuzzer"
-=======
-	      -DSIMDJSON_FUZZ_LDFLAGS=$LIB_FUZZING_ENGINE \
-			  -DSIMDJSON_EXCLUDE_IMPLEMENTATION=haswell
->>>>>>> Kinder, gentler implementation selection
 	
 	ninja all_fuzzers
 	cd ..

